MAINTAINERS

```diff
@@ -300,27 +300,25 @@ or feedback about project operations.
 
 [Org]
     [Org.Curators]
-        People = ["alexellis", "amitshukla", "andrewhsu", "anshulpundir", "chungers", "fntlnz", "gianarb", "olljanat", "programmerq", "ripcurld", "samwhited", "simon-ferquel", "thajeztah"]
+        People = ["alexellis", "amitshukla", "andrewhsu", "anshulpundir", "bsousaa", "chungers", "crazy-max", "fntlnz", "gianarb", "olljanat", "programmerq", "ripcurld", "rumpl", "sam-thibault", "samwhited", "simon-ferquel", "thajeztah", "vvoland"]
     [Org."Docs maintainers"]
-        People = ["chris-crone", "misty", "silvin-lubecki", "simon-ferquel", "thajeztah"]
+        People = ["chris-crone", "silvin-lubecki", "simon-ferquel", "thajeztah"]
     [Org.boot2docker]
         People = ["ehazlett", "tianon"]
     [Org.cli]
-        People = ["albers", "cpuguy83", "silvin-lubecki", "stevvooe", "thajeztah", "tibor", "tonistiigi", "vdemeester", "vieux"]
+        People = ["albers", "cpuguy83", "ndeloof", "rumpl", "silvin-lubecki", "stevvooe", "thajeztah", "tibor", "tonistiigi", "vdemeester", "vieux"]
     [Org.compose]
-        People = ["ndeloof", "rumpl", "ulyssessouza"]
+        People = ["aiordache", "ndeloof", "rumpl", "ulyssessouza"]
     [Org.compose-on-kubernetes]
```

Comment:
The list for compose looks to be outdated; perhaps the repository didn't update with new people, or it's possible it's pulling from the `master` / `main` branch, but the default branch on compose is `v2` (we should change that)